src/main/java/org/stellar/sdk/requests/SSEUtils.java

```diff
@@ -0,0 +1,72 @@
+package org.stellar.sdk.requests;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.internal.sse.RealEventSource;
+import okhttp3.sse.EventSource;
+import okhttp3.sse.EventSourceListener;
+import org.stellar.sdk.responses.GsonSingleton;
+
+import javax.annotation.Nullable;
+import java.net.SocketException;
+
+
+public class SSEUtils {
+  public static <T extends org.stellar.sdk.responses.Response> SSEManager<T> stream(final OkHttpClient okHttpClient, final RequestBuilder requestBuilder, final Class<T> clazz, final EventListener<T> listener) {
```

Comment:
Please rename `clazz` to `responseClass` everywhere.

Reply:
done